llarp/peerstats/peer_db.hpp

```diff
@@ -62,11 +65,27 @@ namespace llarp
     std::optional<PeerStats>
     getCurrentPeerStats(const RouterID& routerId) const;
 
+    /// Configures the PeerDb based on RouterConfig
+    ///
+    /// @param routerConfig
+    void
+    configure(const RouterConfig& routerConfig);
+
+    /// Returns whether or not we should flush, as determined by the last time we flushed and the
+    /// configured flush interval.
+    ///
+    /// @param now is the current[-ish] time
+    bool
+    shouldFlush(llarp_time_t now);
+
    private:
     std::unordered_map<RouterID, PeerStats, RouterID::Hash> m_peerStats;
     std::mutex m_statsLock;
 
     std::unique_ptr<PeerDbStorage> m_storage;
+
+    llarp_time_t m_targetFlushInterval = 30s;
```

Comment:
const?

Reply:
I might make it configurable, but I'll also make it `constexpr` until such time (d88ffbd)